lib/dkim.js

```diff
@@ -463,8 +463,10 @@ function DKIMVerify(email, callback, sigs) {
 
         // The other DNS errors are operational in nature;
         sigs.result = sig.result = false;
-        sigs.issue_desc = sig.issue_desc = err;
-        return callback(err, sigs);
+        sigs.issue_desc = sig.issue_desc = 'There was an error while fetching the DNS record';
+        sigs.issue_desc_detail = sig.issue_desc_detail = [sig.s, sig.d, err.code, err.message].join(' ');
```

Comment:
maybe join with ' | ' or ' -- ' just so we can tell when things are missing? otherwise lgtm